packages/expect/__typechecks__/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "@tsconfig/node10/tsconfig.json",
```

Comment:
could we create a `tsconfig.test.json` in root which has all this stuff instead of repeating in each package?

Reply:
Done. Perhaps `.tsd` pre-suffix is good idea? Also for `jest.config.tsd.js`?